lua/nvim-tree/notify.lua

```diff
@@ -0,0 +1,44 @@
+local notify = {}
+
+local config = {
+  default_level = vim.log.levels.INFO,
+}
+
+local modes = {
+  { name = "trace", level = vim.log.levels.TRACE },
+  { name = "debug", level = vim.log.levels.DEBUG },
+  { name = "info", level = vim.log.levels.INFO },
+  { name = "warn", level = vim.log.levels.WARN },
+  { name = "error", level = vim.log.levels.ERROR },
+}
+
+do
```

Comment:
I would expect to see this block inside the setup function. Is it for the case of notifications before setup?

Please leave a comment explaining.